coalib/output/printers/LogPrinter.py

```diff
@@ -129,14 +133,14 @@ def log_level(self):
         """
         Returns current log_level used in logger.
         """
-        return self.logger.getEffectiveLevel()
+        return self.level
 
     @log_level.setter
-    def log_level(self, log_level):
+    def log_level(self, new_log_level):
```

Comment:
then it belongs into a different commit in theory. For me this isn't more readable, as it's self-explanatory for me that the param for a setter is the *new* value. I'm also used to see `def propname(self, value)` as `propname` normally explains the value itself.
Up to you, but I think this is an unnecessary change, which makes the commit (slightly) bigger